.circleci/config.yml

```diff
@@ -0,0 +1,85 @@
+version: 2
+jobs:
+  lint:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: make lint
+
+  test:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: make test-and-coverage
+
+  build:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - run: make build SLO_EXPORTER_VERSION=${CIRCLE_TAG:-$CIRCLE_SHA1}
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+
+  docker_release:
+    docker:
+      - image: circleci/golang:1.14
+    environment:
+      DOCKERHUB_USERNAME=sklikdevops
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.07.0-ce
+      - attach_workspace:
+          at: .
+      - run:
+          command: |
+            cp build/linux-amd64/slo_exporter ./slo_exporter
+            echo "${DOCKERHUB_PASSWORD}" | docker login -u="${DOCKERHUB_USERNAME}" --password-stdin
+            export SLO_EXPORTER_VERSION=${CIRCLE_TAG:-$CIRCLE_SHA1}
+            make docker-build
+            make docker-push
+
+  github_release:
+    docker:
+      - image: circleci/golang:1.14
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          command: |
+            export SLO_EXPORTER_VERSION=${CIRCLE_TAG:-$CIRCLE_SHA1}
+            go get github.com/github-release/github-release
+            make github-release
```

Comment:
If the make target has some requirements, it should install it itself ideally

Reply:
Fixed in https://github.com/david-vavra/slo-exporter/pull/1/commits/35d37babe7220506c8ffa2b50bf23f2631d79e50